config/tank-maria.json

```diff
@@ -0,0 +1,41 @@
+{
+  "version": 2,
+  "robot": {
+    "modules": {
+      "app": {
+          "driver": "application",
+          "in": ["emergency_stop", "pose2d"],
+          "out": ["desired_speed"],
+          "init": {
+            "max_speed": 0.5
+          }
+      },
+      "robot": {
+          "driver": "osgar.drivers.tankmaria:RobotTankMaria",
```

Comment:
Will we keep the shorthand notation supported by `osgar.drivers.__init__` or are we going to drop it and use this long form everywhere?

Reply:
This branch is relatively old and I did not want to interfere with other code, so that is why long name here. In general I like the short names, so `tankmaria`?

Reply:
Sure, why not.

Reply:
`osgar.drivers.tankmaria:MariaTimer` - this should be some generic timer?? or it can be reprogrammed for input from arduino??

Reply:
I don't see how the last comment relates to the previous discussion on using short names preferably to long ones.